### PR TITLE
cargo-doc: add page

### PR DESCRIPTION
**Pull request by jedahan**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/cargo-doc.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# cargo doc
+
+> Build and view Rust package documentation offline.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-doc.html>.
+
+- Build and view the current packages documentation in your browser:
+
+`cargo doc --open`
+
+- View a particular packages documentation:
```
Comment (zdroid) [inline pages/common/cargo-doc.md]:
```suggestion
- View a particular package's documentation:
```
Is this what you meant?

File: pages/common/cargo-doc.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# cargo doc
+
+> Build and view Rust package documentation offline.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-doc.html>.
+
+- Build and view the current packages documentation in your browser:
```
Comment (zdroid) [inline pages/common/cargo-doc.md]:
```
- Build and view the current package's documentation in your browser:
```

File: pages/common/cargo-doc.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- View all workspace package documentation:
+
+`cargo doc --open --workspace`
+
+- Build documentation without accessing the network:
+
+`cargo doc --offline`
+
+- View a particular dependencies documentation offline:
```
Comment (zdroid) [inline pages/common/cargo-doc.md]: Unclear like the ones above, please rephrase. Do you meant documentation of a particular dependency, or something else?

File: pages/common/cargo-doc.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Build and view the current packages documentation in your browser:
+
+`cargo doc --open`
+
+- View a particular packages documentation:
+
+`cargo doc --open --package {{crate_name}}`
+
+- View all workspace package documentation:
```
Comment (zdroid) [inline pages/common/cargo-doc.md]: Unclear awell. Is it packages related to workspace, or workspace documentation?

**Comment by jedahan** [comment]
After reading the rust documentation more closely, I decided to remove the `--workspace` option since it does nothing unless a package has specified a smaller set of default packages, which is not common. I have also removed the ambiguity between crate and package and just used package for everything, since a package is a superset of crates.

**Comment by jedahan** [comment]
Ahh cool will no longer force push

**Pull request merged**